code/game/objects/structures/tables_racks.dm

```diff
@@ -294,6 +298,59 @@
 	..()
 	return SECONDARY_ATTACK_CONTINUE_CHAIN
 
+/obj/structure/table/CtrlShiftClick(mob/living/user)
+	. = ..()
+	if(!istype(user) || !user.can_interact_with(src) || isobserver(user))
+		return
+	if(!can_flip)
+		return
+	user.balloon_alert_to_viewers("flipping table...")
+	if(!do_after(user, max_integrity * 0.25))
```

Comment:
this is weird, using integrity as a time define. where does it actually come into play?

Reply:
I assume it's meant to model stronger tables = denser tables = harder to flip

Reply:
Yep, it is